src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationCreationRequestValidator.java

```diff
@@ -153,20 +146,40 @@ public boolean isEmptyValue(String value) {
         return isEmpty;
     }
 
-    private Boolean isDxNumberValid(String dxNumber) {
+    private void isDxAddressValid(DxAddressCreationRequest dxAddress) {
+        if (StringUtils.isEmpty(dxAddress.getDxNumber()) && StringUtils.isEmpty(dxAddress.getDxExchange())) {
+            throw new InvalidRequest("Invalid DX Number provided, it cannot be null, empty or greater than 13, and invalid DX Exchange provided, it cannot be null, empty or greater than 20");
+        }
 
-        Boolean numberIsValid = true;
+        if (StringUtils.isEmpty(dxAddress.getDxNumber())) {
+            throw new InvalidRequest("DX Number cannot be null, empty or greater than 13");
```

Comment:
The code is not readable and needs to refactor. So please create jira.

Reply:
Okay, I will create this JIRA. Thanks.